packages/runtime/src/dataViews/DataViewExpander.ts

```diff
@@ -572,9 +572,16 @@ export class DataViewExpander {
 
                 let proposedValueOverruled = false;
                 if (responseItemDVO && responseItemDVO.result === ResponseItemResult.Accepted) {
-                    const proposeAttributeResponseItem = responseItemDVO as ProposeAttributeAcceptResponseItemDVO;
-                    if (JSON.stringify(proposeAttributeResponseItem.attribute.content.value) !== JSON.stringify(proposeAttributeRequestItem.attribute.value)) {
-                        proposedValueOverruled = true;
+                    if (responseItemDVO.type === "AttributeSuccessionAcceptResponseItemDVO") {
+                        const attributeSuccessionResponseItem = responseItemDVO as AttributeSuccessionAcceptResponseItemDVO;
+                        if (JSON.stringify(attributeSuccessionResponseItem.successor.content.value) !== JSON.stringify(proposeAttributeRequestItem.attribute.value)) {
+                            proposedValueOverruled = true;
+                        }
```

Comment:
Use lodash's `isEqual()` instead of JSON.stringify() for deep comparisons of objects. `JSON.stringify()` will break if the order of keys differs in the operands. 

Also, you can yeet the if-clause.

```ts
proposedValueOverrule = !_.isEqual(attributeSuccessionResponseItem.successor.content.value, proposedAttributeRequestItem.attribute.value)
```

Same thing below.